pyproject.toml

```
@@ -46,6 +46,7 @@ starlette-admin = "^0.9.0"
 python-dotenv = "^0.13.0"
 importlib-metadata = {version = "^6.7.0", python = ">=3.7, <3.8"}
 alembic = "^1.11.1"
+distro = "^1.8.0"
```

Not sure if this is worth bringing in a new dependency to the project

---

can we add something like 
```suggestion
distro = {version = "^1.8.0", platform = "linux"}
```

i'm not convinced that `distro` is a heavy dependency, but it's not going to be used at all on the other platforms...